libs/sdk-core/src/moonpay.rs

```diff
@@ -0,0 +1,117 @@
+use anyhow::Result;
+use reqwest::Url;
+
+use crate::breez_services::BreezServer;
+use crate::grpc::SignUrlRequest;
+use crate::SwapInfo;
+
+#[derive(Clone)]
+pub(crate) struct MoonPayConfig {
+    pub base_url: String,
+    pub api_key: String,
+    pub currency_code: String,
+    pub color_code: String,
+    pub redirect_url: String,
+    pub enabled_payment_methods: String,
+}
+
+pub(crate) fn moonpay_config(moonpay_api_key: &str) -> MoonPayConfig {
+    MoonPayConfig {
+        base_url: String::from("https://buy.moonpay.io"),
+        api_key: String::from(moonpay_api_key),
+        currency_code: String::from("btc"),
+        color_code: String::from("#055DEB"),
+        redirect_url: String::from("https://buy.moonpay.io/transaction_receipt?addFunds=true"),
+        enabled_payment_methods: String::from(
+            "credit_debit_card,sepa_bank_transfer,gbp_bank_transfer",
+        ),
+    }
+}
+
+async fn sign_moonpay_url(
```

Comment:
Perhaps rename to `create_moonpay_url` ? This function doesn't really sign.

Reply:
Oh sure, I did move the function to see if the idea works and completely forgot to revisit and refine it.